deploy.py

```diff
@@ -22,9 +23,10 @@ def process(xcode, version):
       new_files = list(filter(lambda x : version in x, new_files))
 
   for i in new_files:
-    print 'Unzip file "{}.zip" to {}'.format(i, target)
+    print('Unzip file "', i, '.zip" to ', target, sep='')
```

Comment:
Why change the code?  Just add the parens: `print('Unzip file "{}.zip" to {}'.format(i, target))` because that is easier to review and is type safe if i or target are ever changed to something other than `str`.